py3status/modules/apt_updates.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+Display number of pending updates for Ubuntu based Distros.
+
+Thanks to Iain Tatch <iain.tatch@gmail.com> for the script that this is based on.
+
+This will display a count of how many 'apt' updates are waiting to be installed.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default 600)
+    format: Display format to use
+        (default 'UPD: {apt}')
+    no_updates: Display format to use if there are no updates
+
+Format placeholders:
+    {apt} Number of pending apt updates
+
+Requires:
+    apt: Needed to display pending 'apt' updates
+
+@author Joshua Pratt <jp10010101010000@gmail.com>
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'UPD: 5'}
+"""
+
+import subprocess
+import sys
+
+FORMAT = 'UPD: {apt}'
+NO_UPDATES = 'UPD'
+LINE_SEPARATOR = "\\n" if sys.version_info > (3, 0) else "\n"
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 600
+    format = FORMAT
```

Comment:
you could just do `format = 'UPD: {apt}'`

`arch_updates` does the odd stuff because of the `aur` stuff.